crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -66,10 +66,15 @@ impl fmt::Debug for RoomEventCache {
     }
 }
 
-/// Thin wrapper for a room event cache listener, so as to trigger side-effects
-/// when all listeners are gone.
+/// Thin wrapper for a room event cache subscriber, so as to trigger
+/// side-effects when all subscribers are gone.
+///
+/// The current side-effect is: auto-shrinking the [`RoomEventCache`] when no
```

Comment:
This side effect happens on `Drop`, right? Might be worth saying that explicitly.

Reply:
Indeed. Addressed!